src/backends/cg.rs

```diff
@@ -308,6 +309,17 @@ impl<D: HasDisplayHandle, W: HasWindowHandle> BufferInterface for BufferImpl<'_,
             }
         };
 
+        // `CGBitmapInfo` consists of a combination of `CGImageAlphaInfo`, `CGImageComponentInfo`
+        // `CGImageByteOrderInfo` and `CGImagePixelFormatInfo` (see e.g. `CGBitmapInfoMake`).
+        //
+        // TODO: Use `CGBitmapInfo::new` once the next version of objc2-core-graphics is released.
+        let bitmap_info = CGBitmapInfo(
+            CGImageAlphaInfo::NoneSkipFirst.0
+                | CGImageComponentInfo::Integer.0
+                | CGImageByteOrderInfo::Order32Little.0
+                | CGImagePixelFormatInfo::Packed.0,
```

Comment:
This was previously `CGBitmapInfo::ByteOrder32Little | CGBitmapInfo(CGImageAlphaInfo::NoneSkipFirst.0)`.

I've added `CGImageComponentInfo::Integer` and `CGImagePixelFormatInfo::Packed` for clarity here, though these are both `0` so they shouldn't affect this functionally.